components/Room.tsx

```diff
@@ -30,6 +29,7 @@ function Room({
   };
   onDisconnected: () => void;
 }) {
+  const { enableExperimentalFeature } = useContext(TelnyxMeetContext);
```

Comment:
Let's rename this to `optionalFeatures` instead.

Reply:
Resolved in [65c2c97](https://github.com/team-telnyx/telnyx-meet/pull/74/commits/65c2c977064f4b714f5e9f3e82165561377b9f69).